homeassistant/components/zwave_js/api.py

```diff
@@ -1713,6 +1713,7 @@ async def websocket_get_config_parameters(
                 "unit": metadata.unit,
                 "writeable": metadata.writeable,
                 "readable": metadata.readable,
+                "default": metadata.default,
```

Comment:
We should update this test and check that the default value is included.

https://github.com/home-assistant/core/blob/86ae70780ccbc58e0314e0f9e2ea26eb8a86d1a9/tests/components/zwave_js/test_api.py#L3048-L3053

This test seems to be using the "Stay awake in battery mode" property value on the Multisensor 6. There the default is 0 (`"default": 0`).